requirements.txt

```diff
@@ -16,6 +16,10 @@ python-socketio==4.4.0
 deepdiff==4.3.1
 python-dateutil==2.8.1
 pid==2.2.5
-azure==4.0.0
+azure-mgmt-compute
```

Comment:
Should probably pin the versions though, like th rest.

Reply:
Thanks for confirming @basnijholt, much appreciated.

@freezeboy, as suggested by Bas,  can you kindly have them pinned so I can merge once all checks passed?

Regards